src/main/java/com/Saurabh/GitHubAiAPI/StringManipulation.java

```diff
@@ -45,4 +53,11 @@ public static String findVowels(String input) {
         }
         return vowels.toString();
     }
+
+    // Helper method to check if a string is a palindrome
+    public static boolean isPalindrome(String input) {
+        String cleanInput = input.replaceAll("\\s+", "").toLowerCase(); // remove spaces and convert to lower case
+        String reversedString = new StringBuilder(cleanInput).reverse().toString();
+        return cleanInput.equals(reversedString);
```

Comment:
The isPalindrome method currently removes spaces and converts to lowercase before checking. This might not be the expected behavior in all use cases. Consider making this behavior optional or creating a separate method for a strict palindrome check.